nsv13/code/modules/overmap/shieldgen.dm

```diff
@@ -200,7 +200,7 @@
 	var/regenPriority = 50
 	var/maxHealthPriority = 50 //50/50 split
 	var/max_power_input = 1.5e+7 //15 MW theoretical maximum. This much power means your shield is going to be insanely good.
-	var/active = FALSE; //Are we projecting out our shields? This lets you offline the shields for a recharge period so that they become useful again.
+	var/active = FALSE; //Are we projecting out our shields? This lets you offline the shields for a recharge period so that they become useful again. This function needs a rework as there is no penalty for shields collapsing, and recharge rate is linear.
```

Comment:
What's with the semicolon?

Reply:
KMC

Reply:
It's a good way to see it needs a rework 🙃